public/app/plugins/datasource/prometheus/specs/datasource_specs.ts

```diff
@@ -606,6 +606,43 @@ describe('PrometheusDatasource', function() {
   });
 });
 
+describe('Prometheus regular escaping', function() {
```

Comment:
These tests should be moved to `datasource.jest.ts` 

we want to move as many tests as possible to jest instead

Reply:
done! :)